static/app/utils/retryableImport.spec.tsx

```diff
@@ -72,12 +72,12 @@ describe('retryableImport', function () {
   it('only retries 3 times', async function () {
     const importMock = jest.fn(
       () =>
-        new Promise((_resolve, reject) => reject(new Error('Loading chunk 123 failed')))
+        new Promise<{default: unknown}>((_resolve, reject) =>
+          reject(new Error('Loading chunk 123 failed'))
+        )
     );
 
-    await expect(retryableImport(() => importMock())).rejects.toThrow(
-      'Loading chunk 123 failed'
-    );
+    await expect(retryableImport(importMock)).rejects.toThrow('Loading chunk 123 failed');
```

Comment:
is this`expect` the same as we use in jest? I didn't know that it could be a `Promise`...interesting

Reply:
![image](https://github.com/getsentry/sentry/assets/29228205/a383aeec-2b40-4161-93f4-ddc243e44c4c)
yes you can ^^